plugins/docker/Dockerfile

```diff
@@ -27,7 +27,7 @@ RUN poetry install --no-dev
 RUN ln -s $(poetry env info -p)/lib/python3.6/site-packages site-packages
 
 
-FROM ghcr.io/hyperledger/aries-cloudagent-python:py3.9-0.9.0
+FROM ghcr.io/hyperledger/aries-cloudagent-python:py3.9-0.10.1
```

Comment:
Unrelated to the PR objective, but noticing this now: base and runtime steps use the same base image, we should be able to use one stage only (I am not sure this is actually providing us with a benefit).

Reply:
Ah, I see - it's to optimize the final image size. We could just uninstall the poetry package after we've completed the build, or leave it as-is. Definitely not a big issue and not something to hold-up this PR.